From 556e10547f3943d2719db390effd7ad00de1bcdf Mon Sep 17 00:00:00 2001
From: Hugues Fruchet <hugues.fruchet@st.com>
Date: Mon, 15 Nov 2021 17:03:41 +0100
Subject: [PATCH 338/529] ARM: configs: multiv7_addons: enable STMIPID02 CSI-2
 camera sensor bridge

Enable STMIPID02 CSI-2 to parallel camera sensor bridge.

Signed-off-by: Hugues Fruchet <hugues.fruchet@st.com>
Change-Id: I2e364fac74d8fe842bb15aa53761c7faa716db11
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/282542
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -65,5 +65,6 @@ CONFIG_UCSI_STM32G0=m
 CONFIG_USB_ONBOARD_HUB=y
 CONFIG_USELIB=y
 CONFIG_VFP=y
+CONFIG_VIDEO_ST_MIPID02=m
 CONFIG_ZBOOT_ROM_BSS=0x0
 CONFIG_ZBOOT_ROM_TEXT=0x0
